src/pages/Agency/Edit/components/AgencySettings/styles.module.scss

```
@@ -1,3 +1,8 @@
+.warning:global(.ant-alert.ant-alert-warning) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)